idea_town/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -1,12 +1,91 @@
 export default `
-  <section class="page" data-hook="experiment-page">
-    <h1>{{title}}</h1>
-    {{#isInstalled}}
-      <button data-hook="uninstall">Uninstall</button>
-    {{/isInstalled}}
-    {{^isInstalled}}
-      <button data-hook="install">Install</button>
-    {{/isInstalled}}
+  <section id="details" class="page" data-hook="experiment-page">
+    <div id="details">
+      <div class="details-sticker"></div>
+
+      <div class="details-header-wrapper">
+        <div class="details-header">
+          <h1>{{title}}</h1>
+          <div class="idea-controls">
+            {{#isInstalled}}
+              <button data-hook="install" class="button primary">Enable {{title}}</button>
+            {{/isInstalled}}
+            {{^isInstalled}}
+              <button data-hook="uninstall" class="button primary">Disable {{title}}</button>
+            {{/isInstalled}}
+            <div class="user-count">7,654,321</div>
+          </div>
+        </div>
+      </div>
+
+      <div class="details-content">
+        <div class="details-overview">
+          <img src="{{thumbnail}}" width="260" height="auto">
```

Comment:
Hmm, yeah I think you can specify that in CSS, but not in markup. But I think if you leave the height out, I think it will auto scale in correct aspect ratio to width